.github/actions/appimage/action.yml

```diff
@@ -0,0 +1,55 @@
+name: Create and upload AppImage
+description: "Create and upload AppImage as a Github Actions artifact."
+inputs:
+  use_qt6:
+    description: "Specify whether we're using Qt 5 or Qt 6"
+    default: ON
+
+runs:
+  using: "composite"
+  steps:
+    - name: Build
+      run: cmake --build '${{github.workspace}}/build' --parallel "$(nproc)"
+      shell: bash
+
+    - name: Install
+      run: cmake --install '${{github.workspace}}/build'
+      shell: bash
+
+    - name: Get app version
+      run: |
+        if [[ "${{ inputs.use_qt6 }}" = "ON" ]]; then
+          VERSION=qt6-
+        else
+          VERSION=qt5-
+        fi
+        VERSION+="$(grep APP_VERSION quickevent/app/quickevent/src/appversion.h | cut -d\" -f2)"
+        echo "VERSION=$VERSION" >> "$GITHUB_ENV"
+      shell: bash
+
+    - name: Get linuxdeploy and linuxdeploy-plugin-qt
+      run: |
+        curl -JOL "https://github.com/linuxdeploy/linuxdeploy/releases/download/continuous/linuxdeploy-x86_64.AppImage"
+        curl -JOL "https://github.com/linuxdeploy/linuxdeploy-plugin-qt/releases/download/continuous/linuxdeploy-plugin-qt-x86_64.AppImage"
```

Comment:
to my knowledge `linuxdeploy-plugin-qt-x86_64.AppImage` is equal to `linuxdeploy-x86_64.AppImage --plugin qt`  
is it necessary to download both?

Reply:
It is necessary, `linuxdeploy-x86_64.AppImage --plugin qt` invokes `linuxdeploy-plugin-qt-x86_64.AppImage`.

Reply:
You are right, I have mistaken `linuxdeploy` for [linuxdeployqt](https://github.com/probonopd/linuxdeployqt)